exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,8 @@
 // Start by creating a variable `message`
 
 console.log(message);
+const name1 = " Danial";
+const message = name1.trim();
+var nameLength = name1.length;
+var message = "My  name is" + trimName + " and my name is " +nameLength +"character long"
```

Comment:
Make sure that you're consistent with spaces! There should always be a space after the `+` operator.